packages/multi-trait-rubric/src/trait.jsx

```diff
@@ -1,67 +1,83 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import {withStyles} from '@material-ui/core/styles';
+
+const styles = () => ({
+    trait: {
+        '&$traitName': {
+            color: '#7E8494',
+            verticalAlign: 'middle'
+        }
+    },
+    noDescription: {
+        color: '#F1F1F1',
+        textAlign: 'center'
+    },
+    traitName: {}
+});
 
 const Trait = (props) => {
-  const { trait, traitIndex, scaleIndex, showStandards, showDescription, scorePointsValues } = props;
-  const { name, standards, scorePointsDescriptors, description } = trait || {};
+    const {trait, traitIndex, scaleIndex, showStandards, showDescription, scorePointsValues, classes} = props;
+    const {name, standards, scorePointsDescriptors, description} = trait || {};
 
-  return (
-    <tr key={`scale-${scaleIndex}-trait-${traitIndex}`}>
-      <td>
-        <div dangerouslySetInnerHTML={{ __html: name }}/>
-      </td>
+    return (
+        <tr key={`scale-${scaleIndex}-trait-${traitIndex}`}>
+            <td className={`${classes.trait} ${classes.traitName}`}>
+                <div dangerouslySetInnerHTML={{__html: name}}/>
+            </td>
 
-      {showStandards
-        ?
-        <td>
-          <div dangerouslySetInnerHTML={{ __html: standards.join(',') }}/>
-        </td>
-        : null
-      }
+            {showStandards
```

Comment:
you formatting 4 spaces? should be 2 spaces (no tabs!)

Reply:
will fix